src/bot/handlers/config.py

```diff
@@ -1,20 +1,21 @@
-from .utils import clean_config_data
 from .filters import private_text_filter
+from .utils import clean_config_data, enumerate_options
 from telegram import ReplyKeyboardMarkup, ReplyKeyboardRemove
 from telegram.ext import CommandHandler, Filters, ConversationHandler, MessageHandler
 
 # Messages
-NO_CONFIG       = "You don't have any chat to configure.\nYou need to use /create command in some chat first."
-SELECT          = 'Select the chat that you want to configure'
-OPTIONS         = 'Start writing the options one by one.\nAdditionally you can use /del to delete some options, or /add to continue adding.\nUse /done at the end'
-WRONG_CHAT      = "You don't have any configuration active in the selected chat, sorry :(, try /config again."
-INVALID_OPTION  = "I don't have this option, select a valid one"
-EMPTY           = "You don't have options to delete"
-CHOOSE_DEL      = 'Choose the options to delete one by one'
-USELESS         = "You didn't provide any options, try /config again when you are ready"
-DONE_CONFIG     = 'Perfect! The options provided by you are not editable.\nType /close in the related chat in order to close the discussion'
-INIT_DISCUSS    = 'Time to vote!!!\nSend /vote to participate.\n\nThe options to organize are:\n%s'
-BYE             = 'See you soon.'
+NO_CONFIG       = "Usted no tiene ningun chat para configurar.\nNecesita usar el comando /create en algun chat."
+SELECT          = 'Seleccione el chat que desea configurar'
+OPTIONS         = 'Comience a escribir las opciones en mensajes separados cada una. Puede utilizar los siguientes 3 comandos auxiliares durante la configuración.\n- /del Para eliminar algunas opciones\n- /add Para continuar añadiendo opciones\n- /done Para guardar la configuración'
+WRONG_CHAT      = "Usted no tiene acceso a la configuración del chat que selecciono :(, utilice /config nuevamente y seleccione algun chat valido."
+INVALID_OPTION  = "Ha seleciona una opción desconocida"
+EMPTY           = "La lista de opciones esta vacia"
```

Comment:
Faltan las tildes en `está` y en `vacía`.

Reply:
That will be fixed in #23